doc/api/esm.md

```diff
@@ -741,61 +727,54 @@ changes:
     description: Add support for chaining resolve hooks. Each hook must either
       call `nextResolve()` or include a `shortCircuit` property set to `true`
       in its return.
-  - version:
-    - v17.1.0
-    - v16.14.0
     pr-url: https://github.com/nodejs/node/pull/40250
     description: Add support for import assertions.
 -->
 
 > The loaders API is being redesigned. This hook may disappear or its
 > signature may change. Do not rely on the API described below.
 
-* `specifier` {string}
-* `context` {Object}
-  * `conditions` {string\[]} Export conditions of the relevant `package.json`
-  * `importAssertions` {Object}
+  * `importAssertions` {Object} The object after the `assert` in an `import`
+    statement, or the value of the `assert` property in the second argument of
+    an `import()` expression; or an empty object
```

Comment:
```suggestion
  * `importAssertions` {Object} An object whose key+value pairs represent the
    assertions for the module to import
```

Reply:
I’m not sure this is any better. Can we call this a wrap for now? I’ve pushed a lot of changes.

Reply:
The current text is technically wrong, which I don't think is acceptable for a technical documentation. I'm not found of my version, but I'd rather leave the description empty that fill it in with something incorrect, so that's also an option if you prefer.

> I’ve pushed a lot of changes.

Yeah what's up with that? It looks like a lot of unrelated changes have made their way to this PR, making the linter unhappy. May suggest to use fixup commits instead of force pushing to simplify reviews and limit the risk of such mistake to slip through?

Reply:
I don’t know what happened with `esm.md`, it’s gotten destroyed somehow. Working on it.